lib/reservation_monitor.py

```diff
@@ -196,6 +196,7 @@ def _get_reservations(self) -> Tuple[List[Dict[str, Any]], bool]:
         because new headers might not be valid and a list of reservations could not be retrieved.
         """
         logger.debug("Retrieving reservations for account")
+        time.sleep(1)
```

Comment:
@dmytrokoren is this supposed to be here?

Reply:
Hey @dmytrokoren, any follow up to this? This PR will be merged once this conversation gets resolved.